### PR TITLE
Add travis

### PR DESCRIPTION
**Pull request by alorma**
Because... why not?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: java
+jdk:
+  - oraclejdk7
+  - openjdk7
+
+before_install:
+ - chmod +x gradlew
```
Comment (Sloy) [inline .travis.yml]: Why this line? I believe the file already has execution permissions.

Comment (alorma) [inline .travis.yml]: Ups, i copied from another travis file.

But... i will remove if you want, but i think it's not a bad idea to make it executable, what if it changed somewhen?

Comment (alorma) [inline .travis.yml]: Done

Comment (Sloy) [inline .travis.yml]: Don't let a windows user touch your repo :P

I'd like the build  to fail it anybody changes the execution permission, they would make my life harder

**Comment by alorma** [comment]
https://travis-ci.org/alorma/gallego/builds

**Comment by Sloy** [comment]
Awesome! Thanks @alorma ❤️

**Pull request merged**